tslint.json

```diff
@@ -8,7 +8,12 @@
     "interface-name": [true, "never-prefix"],
     "no-shadowed-variable": false,
     "no-string-literal": false,
-    "trailing-comma": false,
-    "member-ordering": ["fields-first"]
+    "trailing-comma": [false],
+    "member-ordering": ["fields-first"],
+    "array-type": ["array-simple"],
+    "arrow-parens": false,
+    "max-classes-per-file": [false],
+    "prefer-for-of": false,
+    "ban-types": false

```

Comment:
I think you can use `fn :() => void` in place of `fn: Function`. Of course, the more specific we can be about the function signature, the better. `Function` kind of acts like `any`- I think that's one reason it's "banned" by default.